test/index.spec.ts

```diff
@@ -61,10 +61,6 @@ const fixtureFilePath = (ruleSet: string): string => {
     return `${BASE_PATH}/eslint/`;
   }
 
-  if (ruleSet === 'overrides') {
```

Comment:
I removed the `overrides` section in the previous PR, forgot to remove this part